source/unified-test-format/unified-test-format.rst

```diff
@@ -3,13 +3,13 @@ Unified Test Format
 ===================
 
 :Spec Title: Unified Test Format
-:Spec Version: 1.7
+:Spec Version: 1.8
```

Comment:
Note: https://github.com/mongodb/specifications/pull/1188 is also bumping the schema version so depending on which PR gets merged first the other will need to rebase accordingly.